roles-in-tech/developer-advocate.md

```diff
@@ -78,7 +83,7 @@ Outbound tasks
 - Participate and help with events
 - Act as a “firewall” for internal teams
 - Help dealing with influencers
-
+  ##### Explanation
 Be a social media presence:
 Provide a personal channel to ask about your company or products. People react much better to a human and interact more with them than with a corporate account. Be yourself — at no point you should become a corporate sales person. Do triage — point people to resources to use like feedback channels instead of talking on their behalf. Don’t make decisions for your colleagues. Help them, don’t add to their workload.
```

Comment:
For both explanation sections:  please split these ideas up into paragraphs or bullet points.  in the code, it may look like it's separated, but in the markdown preview, it is all one huge block of text.  Be sure to look at the preview to see the final results.